docs/release-notes/1.5.3/release-1.5.3.md

```diff
@@ -0,0 +1,41 @@
+# [ML.NET](http://dot.net/ml) 1.5.3
+
+## **New Features**
+- **New API for exporting models to Onnx**. ([#todo](https://github.com/dotnet/machinelearning/pull/todo)). A new API has been added to Onnx converter to specify the output columns you care about. This will export a smaller and more performant model in many cases.
+
+## **Enhancements**
+- Perf improvement for TopK Accuracy and return all topK in Classification Evaluator ([#5395](https://github.com/dotnet/machinelearning/pull/5395))
```

Comment:
This is fine as is, though it could be listed under the New Features section, since it adds a new multi-class metric. There's a new array output which returns _all_ Top-K Accuracies instead of just a singular.

The speed benefit is mainly for users who called the eval function K times to get all K values; this reduces it to just once call by calculating all at once, and returning all as a new metric (while keeping the old singular valued metric).

Fine line between an enhancement or new feature.

Reply:
It is a fine line. :-) I think I will keep it as is. 

---
In reply to: [539738568](https://github.com/dotnet/machinelearning/pull/5543#discussion_r539738568) [](ancestors = 539738568)